src/builder.rs

```diff
@@ -152,6 +154,14 @@ impl TunBuilder {
         self
     }
 
+    /// Sets CLOEXEC flag on all tun/tap FDs. This will prevent the fds from leaking to any exec-ed
+    /// child processes.
+    /// Defaults to true.
+    pub fn set_cloexec(mut self, set_cloexec: bool) -> Self {
```

Comment:
To align with other boolean-based parameters, this method should avoid taking a bool as input. If the default value is `true`, then: 

```rust
pub fn no_close_on_exec(mut self) -> Self {
    self.cloexec = false;
    self
}
```

Otherwise:

```rust
pub fn close_on_exec(mut self) -> Self {
    self.cloexec = true;
    self
}
```